DEVELOPMENT.md

```diff
@@ -38,15 +38,15 @@ hatch run all:test
 
 ## Submitter environment
 
-Cinema4D does not support PYTHONPATH. We set DEADLINE_CLOUD_PYTHONPATH which the
-submitter and adaptor uses to set sys.path explictly and load deadline modules.
+Cinema4D does not support PYTHONPATH. We set CINEMA4D_DEADLINE_CLOUD_PYTHONPATH which the
+submitter uses to set sys.path explictly and load deadline modules.
 
 - install deadline-cloud with pyside
 - set env below
 
 ```
 # deadline-cloud lib with pyside
-export DEADLINE_CLOUD_PYTHONPATH="/path/to/deadline-cloud/site-packages"
```

Comment:
Hmm, I was reviewing a [different PR](https://github.com/aws-deadline/deadline-cloud-for-cinema-4d/pull/73#discussion_r1815368314) where a similar conversation came up. Turns out Cinema 4D already provides a way to register custom interpreter bound search paths, similar to the PYTHONPATH environment variable.
 
So I think we may not need these changes.

Reply:
Yea, we shouldn't need this based off the other changes that leverage loading C4D

Reply:
Good catch!

I will keep an eye out for #73 to get merged in and rebase out this change.